### ISSUE TITLE
Footnote target unclear in many places

### ISSUE THREAD

**Issue by pgmmpk** [slavonic/cu-books#16]
From Dobrotolyubie `1_01.xml`:
```
<p>рл҃и. Сме́ртное безсмє́ртнымъ подлежи́тъ, безсме́ртное же 
смє́ртнымъ:<footnote lang="cu">Безсме́ртное здѣ́сь за долговре́менное и҆ 
долгопребыва́ющее бере́тсѧ.</footnote> слꙋ́жатъ, си́рѣчь, стїхі̑и человѣ́кꙋ, 
по человѣколю́бїю, и҆ врожде́нной въ ни́хъ бла́гости созда́вшагѡ бг҃а.</p>
```

Or `2_03.xml`:
```
...Поне́же ᲂу҆́бѡ по писа́нїю, всѧ́ко дре́во, не творѧ́щее плода̀ добра̀ 
посѣка́етсѧ, и҆ во ѻ҆́гнь вмета́етсѧ:<footnote lang="cu">Лꙋкѝ гл. г҃, ст. ѳ҃.</footnote> 
ꙗ҆́вѣ, ꙗ҆́кѡ ѡ҆ плода́хъ всѐ на́ше тща́нїе до́лженствꙋетъ...
```

It is hardly reasonable to use the last word of the preceding phrase as footnote target. Especially when target is Jesus' parable.

How the original renders this? As a numbered superscript? Maybe we should leave superscript and mark this up as `<sup>1</sup><footnote...`?

**Comment by typiconman** [comment]
I think these need to be reviewed manually. They're probably only in Dobrotolyubie, which is a bit special, since it's not a liturgical text.

**Comment by typiconman** [comment]
Is there a way to generate a list of questionable footnotes that need to be reviewed? Perhaps all footnotes that don't have a kavyka or an asterisk nearby?

**Comment by pgmmpk** [comment]
Yes, I have scripts here to generate Markdown. Footnote target is being searched for with regexes. When regex can not find target, this should be a red flag.

I will post some questionable footnotes to review when get some time. No point in reviewing everything.
